prow/scripts/cluster-integration/kyma-integration-gardener.sh

```diff
@@ -103,25 +103,22 @@ gardener::generate_overrides
 export CLEANUP_CLUSTER="true"
 gardener::provision_cluster
 
-if [[ "${KYMA_MAJOR_VERSION}" == "2" ]]; then
+kyma::deploy_kyma \
+  -p "$EXECUTION_PROFILE" \
+  -d "$KYMA_SOURCES_DIR"
+if [[ "${KYMA_DELETE}" == "true" ]]; then
+  sleep 30
+  kyma::undeploy_kyma
+  sleep 30
   kyma::deploy_kyma \
-    -p "$EXECUTION_PROFILE" \
-    -d "$KYMA_SOURCES_DIR"
-  if [[ "${KYMA_DELETE}" == "true" ]]; then
-    sleep 30
-    kyma::undeploy_kyma
-    sleep 30
-    kyma::deploy_kyma \
-       -p "$EXECUTION_PROFILE" \
-       -d "$KYMA_SOURCES_DIR"
-  fi
+      -p "$EXECUTION_PROFILE" \
+      -d "$KYMA_SOURCES_DIR"
+fi
 # this will be extended with the next components
-elif [[ "${API_GATEWAY_INTEGRATION}" == "true" ]]; then
+if [[ "${API_GATEWAY_INTEGRATION}" == "true" ]]; then
   api-gateway::prepare_components_file
   integration_tests::install_kyma
   api-gateway::deploy_login_consent_app
```

Comment:
Should these 3 functions be called for Kyma 2?
In previous version of the script this `elif` was corresponding to the `if [[ "${KYMA_MAJOR_VERSION}" == "2" ]]`

Reply:
This is OK, since the API-gateway pipeline doesn't use the **kyma_major_version_2** config, and the API_GATEWAY_INTEGRATION env variable is only used for this job

Reply:
Yes, it seems that the logic should be inverted, with first checking whether `API_GATEWAY_INTEGRATION == true` and `else` doing the logic that was under `if [[ "${KYMA_MAJOR_VERSION}" == "2" ]]`

Reply:
fixed.